panc-docs/source/pan-book/pan-book.rst

```diff
@@ -815,6 +815,29 @@ included templates. Although multiple ``prefix`` statements can be used
 in a template, it is best practice to use only one near the beginning of
 the template.
 
+The ``prefix`` statement itself also supports a relative path. In that case,
+a previously defined ``prefix`` statement with an absolute path is required,
+and the relative prefix is relative to the (last) absolute prefix.
+This is very useful when dealing with very large paths and a complex
+schema (e.g. ``ncm-metaconfig`` configuration,
```

Comment:
I would not reference configuration modules in the panc documentation. May be just done an example of a long path.

Reply:
hmmm, i'd rather drop the example in that case.